mappings/com/mojang/realmsclient/network/Slot.mapping

```diff
@@ -0,0 +1,39 @@
+CLASS net/minecraft/class_4883 com/mojang/realmsclient/network/Slot
+	FIELD field_22632 pvp Ljava/lang/Boolean;
+	FIELD field_22633 spawnAnimals Ljava/lang/Boolean;
+	FIELD field_22634 spawnMonsters Ljava/lang/Boolean;
+	FIELD field_22635 spawnNpcs Ljava/lang/Boolean;
+	FIELD field_22636 spawnProtection Ljava/lang/Integer;
+	FIELD field_22637 commandBlocks Ljava/lang/Boolean;
+	FIELD field_22638 forceGameMode Ljava/lang/Boolean;
+	FIELD field_22639 difficulty Ljava/lang/Integer;
+	FIELD field_22640 gameMode Ljava/lang/Integer;
+	FIELD field_22641 slotName Ljava/lang/String;
+	FIELD field_22642 worldTemplateId J
+	FIELD field_22643 worldTemplateImage Ljava/lang/String;
+	FIELD field_22644 adventureMap Z
+	FIELD field_22645 empty Z
+	FIELD field_22646 DEFAULT_WORLD_TEMPLATE_IMAGE Ljava/lang/String;
+	METHOD <init> (Ljava/lang/Boolean;Ljava/lang/Boolean;Ljava/lang/Boolean;Ljava/lang/Boolean;Ljava/lang/Integer;Ljava/lang/Boolean;Ljava/lang/Integer;Ljava/lang/Integer;Ljava/lang/Boolean;Ljava/lang/String;)V
+		ARG 1 pvp
+		ARG 2 spawnAnimals
+		ARG 3 spawnMonsters
+		ARG 4 spawnNPCs
```

Comment:
`spawnNpcs` to be consistent with the field above

Reply:
if (!this.spawnNPCs) {
      jsonObject2.addProperty("spawnNPCs", this.spawnNPCs);
}